core/storage/blog/gae_models.py

```diff
@@ -47,7 +47,9 @@ class BlogPostModelDataDict(TypedDict):
 
 
 class BlogPostModel(base_models.BaseModel):
-    """Model to store blog post data.
+    """Model to store blog post data.Functionality to allow authors to revert
```

Comment:
Add a space after the period.

Could you please explain briefly why this functionality is not being built in here (what is the rationale)? I think that would be helpful.

Reply:
Added. We don't want to maintain commit history for the blog post models, therefore, we did not find it necessary to support reverting back to previous versions of the blog post and thus having versioned BlogPostModel.  //cc @iamprayush